tests/ada_grad_test.cpp

```diff
@@ -1,8 +1,6 @@
 /**
- * @file ada_grad_test.cpp
- * @author Abhinav Moudgil
+ * @file ada_sqrt_test.cpp
```

Comment:
Oops, did you mean to commit these changes to `ada_grad_test.cpp` or just to the new test file?